### ISSUE TITLE
Support jekyll version 4.0.0

### ISSUE THREAD

**Issue by sai-manoj-kumar** [pages-themes/hacker#52]
When I run jekyll new <sitename>, I get Gemfile with jekyll version of 4.0.0. The latest release of this theme still supports only upto 3.5. Can this be updated to work with 4.0.0?

**Comment by tanant** [comment]
@sai-manoj-kumar it looks like the theme itself works fine. This looks like the same issue in #44 where the repo has a relaxed set of restrictions, but the gemfile is only released to 0.1.1 which is before this. In the short term, depending on what you want to do you could update your Gemfile to directly point at this repo:
```
gem "jekyll-theme-hacker", git: 'https://github.com/pages-themes/hacker'
```
or do a local build

**Comment by parkr** [comment]
0.1.2 is released: https://github.com/pages-themes/hacker/releases/tag/v0.1.2
